src/modules/organizations/containers/collaborators-container.jsx

```diff
@@ -226,12 +229,26 @@ class CollaboratorsContainer extends React.Component {
   }
 }
 
+CollaboratorsContainer.defaultProps = {
+  location: {},
+  organization: {},
+  organizationCollaborators: [],
+  organizationOwner: {},
+  user: {},
+};
+
 CollaboratorsContainer.propTypes = {
-  children: React.PropTypes.node,
   dispatch: React.PropTypes.func,
+  location: React.PropTypes.shape({
+    query: React.PropTypes.object,
+    page: React.PropTypes.number,
```

Comment:
The linter throws `'location.query.page' is missing in props validation` if `page` is left out. I changed it to a string, thanks for pointing that out!